src/sentry/api/endpoints/project_stacktrace_links.py

```diff
@@ -0,0 +1,148 @@
+from __future__ import annotations
+
+from rest_framework import serializers
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry.api.base import region_silo_endpoint
+from sentry.api.bases.project import ProjectEndpoint
+from sentry.integrations.base import IntegrationInstallation
+from sentry.integrations.mixins import RepositoryMixin
+from sentry.models import Project, RepositoryProjectPathConfig
+from sentry.services.hybrid_cloud.integration import integration_service
+from sentry.shared_integrations.exceptions import ApiError
+
+from .project_stacktrace_link import get_code_mapping_configs
+
+MAX_CODE_MAPPINGS_USED = 3
```

Comment:
just curious - how was this number chosen?

Reply:
It looks like most project only need to use a single code mapping to resolve all the files (since it's all in the same repo), sentry for example requires 2 (sentry + getsentry). So I just chose 3 as the max since that should cover most of the use case. I'll add some metrics to track this.